.gitignore

```diff
@@ -41,3 +41,7 @@ version_local.json
 
 # Output files
 out
+
+# Rust
+iree/turbine/kernel/wave/scheduling/aplp/target/
+iree/turbine/kernel/wave/scheduling/aplp/Cargo.lock
```

Comment:
I would scope this to https://github.com/iree-org/iree-turbine/tree/main/iree/turbine/kernel/wave or another of those subfolders by adding a new .gitignore there. We might move kernel/wave to another repository at some point and keeping subfolder-specific rules local to the subfolder will make that easier.

Reply:
I added a standard python gitignore to wave and moved the rust related lines.